node_modules/.bin/loose-envify

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
```

Comment:
Please add node_modules in .gitignore.

Reply:
Added node_modules in gitignore and deleted it from the repo